### PR TITLE
Initial newsletter draft

### PR DESCRIPTION
**Pull request by ericholscher**
Needs a bit more content.

### CODE REVIEW

File: newsletter-april-2022.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+   :tags: newsletter, python
+   :author: Eric Holscher
+   :location: BND
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - April 2022
```
[the remark this answers is not in the document]
Comment (humitos) [reply newsletter-april-2022.rst:11]: Also, we can add here that we are now linking (and force validation in .com) the `RemoteRepository` when importing a project via the API: https://github.com/readthedocs/readthedocs.org/pull/8983

**Review by humitos — COMMENTED** [review]
LGTM! I added some of the things that I remembered we have done this month as well that could be added to the newsletter.

File: newsletter-april-2022.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+We plan to share this on a few job boards,
+and are looking for someone to join the team who is excited to work on our product.
+
+New features
+------------
+
+In March we shipped a long requested feature,
+so I won't bury the lede:
+
+- **We now allow custom command to run before and after your builds**: ``build.jobs``. This is a long requested feature, so `check it out`_ now!
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply newsletter-april-2022.rst]: @humitos Nope, you're right. Was a typo :)

**Review by ericholscher — COMMENTED** [review]
Updated!

**Review by humitos — APPROVED** [review]
Great updates! 💪

**Review by agjohnson — APPROVED** [review]
Looks great!

**Pull request merged**